x/staking/legacy/v040/migrate.go

```diff
@@ -130,6 +130,7 @@ func Migrate(stakingState v038staking.GenesisState) *GenesisState {
 			HistoricalEntries: uint32(stakingState.Params.HistoricalEntries),
 			BondDenom:         stakingState.Params.BondDenom,
 			MinCommissionRate: v040staking.DefaultMinCommissionRate,
+			// MinSelfDelegation: v040staking.DefaultMinSelfDelegation,
```

Comment:
Is there a reason this was added?

Reply:
Ah looks like I forgot to remove this comment - I had mistakenly tried to add support for the new parameter to the migration file based on previous commits for other parameters that were added and realized afterwards that that was not necessary/correct. Removed in my most recent commit!